jenkins-pipelines/alternator/longevity-alternator-200gb-48h.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    params: params,
+
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: 'test-cases/longevity/longevity-alternator-200GB-48h.yaml',
+
+    timeout: [time: 3000, unit: 'MINUTES'],
+    email_recipients: '["qa@scylladb.com", "alternator@scylladb.com"]'
```

Comment:
this is the place where we saw emails not being sent to qa list... do we have an issue for it?

Reply:
could be the our support for multiple mails is a bit off, we'll need to look into the code

Reply:
Issue was address, was a problem in the pipeline code